metals/src/main/scala/scala/meta/internal/metals/BloopInstall.scala

```diff
@@ -214,6 +214,8 @@ object BloopInstall {
   private class ProcessHandler() extends NuAbstractProcessHandler {
     var response: Option[CompletableFuture[_]] = None
     val completeProcess = Promise[BloopInstallResult]()
+    val standardOutput = new StringBuilder
+    val errorOutput = new StringBuilder
 
     override def onStart(nuProcess: NuProcess): Unit = {
```

Comment:
Should we flush the buffer on exit?

Reply:
Which buffer? We just use the logger and pretty sure it has autoflush set.

Reply:
What happens in the following scenario?

- stdout prints `"Hello"`, with no newline
- process exits

I expect `"Hello"` to get printed out on exit.

Reply:
But that is not going to happen, we only run build tools using this class.  

I can account for scenario, but it seems really hypothetical.

Reply:
Ok, added additional logging on exit.